lib/Ed25519Signature2020.js

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import jsonld from 'jsonld';
+import jsigs from 'jsonld-signatures';
+const {suites: {LinkedDataSignature}} = jsigs;
+const base58btc = require('base58-universal');
+import {
+  Ed25519VerificationKey2020
+} from '@digitalbazaar/ed25519-verification-key-2020';
+
+export class Ed25519Signature2020 extends LinkedDataSignature {
+  /**
+   * @param {object} options - Options hashmap.
+   * @param {string} [options.verificationMethod] - A key id URL to the paired
+   *   public key.
+   * @param {Function} [options.signer] - Signer function.
+   *
+   * Advanced optional parameters and overrides:
+   *
+   * @param {object} [options.key] - An optional crypto-ld KeyPair.
+   * @param {object} [options.proof] - A JSON-LD document with options to use
+   *   for the `proof` node (e.g. any other custom fields can be provided here
+   *   using a context different from security-v2).
+   * @param {string|Date} [options.date] - Signing date to use if not passed.
+   * @param {boolean} [options.useNativeCanonize] - Whether to use a native
+   *   canonize algorithm.
+   */
+  constructor({
+    signer, verificationMethod, key, proof, date, useNativeCanonize
+  } = {}) {
+    super({
+      type: 'https://w3id.org/security#Ed25519Signature2020',
+      alg: 'EdDSA', LDKeyClass: Ed25519VerificationKey2020, verificationMethod,
+      signer, key, proof, date, useNativeCanonize
+    });
+    this.requiredKeyType = 'Ed25519VerificationKey2020';
+
+    if(key) {
+      if(verificationMethod === undefined) {
+        this.verificationMethod = key.id;
+      }
+      this.key = key;
+      if(typeof key.signer === 'function') {
+        this.signer = key.signer();
+      }
+      if(typeof key.verifier === 'function') {
+        this.verifier = key.verifier();
+      }
+    }
+  }
+
+  /**
+   * Adds a signature (proofValue) field to the proof object. Called by
+   * LinkedDataSignature.createProof().
+   *
+   * @param {Uint8Array} verifyData - Data to be signed.
+   * @param {object} proof - Proof object (containing the proofPurpose,
+   *   verificationMethod, etc).
+   *
+   * @returns {Promise<object>} Resolves with the proof containing the signature
+   *   value.
+   */
+  async sign({verifyData, proof}) {
+    if(!(this.signer && typeof this.signer.sign === 'function')) {
+      throw new Error('A signer API has not been specified.');
+    }
+    const signatureBytes = await this.signer.sign({data: verifyData});
+
+    // prefix with `z` to indicate multi-base base58btc encoding
+    proof.proofValue = `z${base58btc.encode(signatureBytes)}`;
```

Comment:
I'm not sure if this is the right place to do this off-hand or not, but you could try this:

```suggestion
    proof.proofValue = {'@value': `z${base58btc.encode(signatureBytes)}`, '@type': <put multibase URL from context here>'};
```

Reply:
Trying it out. 

Setting that line to:

```js
    // prefix with `z` to indicate multi-base base58btc encoding
    proof.proofValue = {'@value': `z${base58btc.encode(signatureBytes)}`,
      '@type': 'https://w3id.org/security#multibase'};
```
yields this error:

```
     jsonld.SyntaxError: Invalid JSON-LD syntax; "@graph" value must not be an object or an array.
      at _expandObject (node_modules/jsonld/lib/expand.js:566:13)
      at Object.api.expand (node_modules/jsonld/lib/expand.js:248:9)
      at _expandObject (node_modules/jsonld/lib/expand.js:795:35)
      at api.expand (node_modules/jsonld/lib/expand.js:248:9)
      at Function.jsonld.expand (node_modules/jsonld/lib/jsonld.js:330:24)
      at Function.jsonld.compact (node_modules/jsonld/lib/jsonld.js:173:29)
      at ProofSet.add (node_modules/jsonld-signatures/lib/ProofSet.js:114:41)
      at async Proxy.sign (node_modules/jsonld-signatures/lib/jsonld-signatures.js:55:12)
      at async Context.<anonymous> (test/Ed25519VerificationKey2020.spec.js:51:22)
```